src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
-        "Air Elemental",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#descriptiveCard#"
+    ],
+
+    "descriptiveCard": [
+        "an altered #realCard#",
+        "an artist-signed #realCard#",
+        "an HP coffee-stained #realCard#",
+        "a counterfeit #realCard#",
+        "a foil #realCard#",
+        "a foreign-language #realCard#",
+        "a From the Vaults #realCard#",
+        "a proxy #realCard#",
+        "#realCard.a# borrowed from a dealer",
+        "#pro#'s personal #realCard#"
+    ],
+
+    "cardAlone": [
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#fakeCard#"
+    ],
+
+    "realCard": [
```

Comment:
Ah, I see what you mean. There's so many more normal cards than either great or terrible ones, though, that any other weighting would see a lot more repeats. Even as is, you'll see any given great/terrible card far more often than any given average one.